clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -154,4 +158,8 @@ class Keys {
   static String getImageKey(String imageId, String account, String region) {
     "${ID}:${Namespace.IMAGES}:${account}:${region}:${imageId}"
   }
+
+  static String getInstanceTypeKey(String instanceType, String account, String region) {
+    "${ID}:${Namespace.INSTANCE_TYPES}:${instanceType}:${account}:${region}"
```

Comment:
We seem to be inconsistent with order in these keys. I don't think it is a problem, just noting it.